src/common/GroupAvatar.js

```diff
@@ -53,17 +56,17 @@ export default class GroupAvatar extends PureComponent<Props> {
       height: size,
       width: size,
       borderRadius: size / 8,
-      backgroundColor: colorHashFromString(names.join(', ')),
-    };
-    const textSize = {
-      fontSize: size / 3,
-      color: foregroundColorFromBackground(frameSize.backgroundColor),
+      backgroundColor: Color(colorHashFromString(names.join(', ')))
+        .lighten(0.6)
+        .hex(),
     };
 
+    const iconColor = Color(colorHashFromString(names.join(', '))).string();
+
     return (
       <Touchable onPress={onPress}>
         <View style={[styles.frame, frameSize]}>
-          <Text style={[styles.text, textSize]}>{initialsForGroupIcon(names)}</Text>
```

Comment:
`styles.text`, defined near the top of the file, isn't needed anymore and should be removed in this commit.